plugins/catalog-import/src/components/StepPrepareCreatePullRequest/StepPrepareCreatePullRequest.tsx

```diff
@@ -108,7 +114,11 @@ export const StepPrepareCreatePullRequest = ({
     });
 
     // TODO: defaultKind (=group), defaultNamespace (=same as entity)
-    return groupEntities.items.map(e => serializeEntityRef(e) as string).sort();
+    const groupOptions = groupEntities.items
+      .map(e => serializeEntityRef(e) as string)
+      .sort();
+
+    return [...[CODEOWNER_GROUP_REF], ...groupOptions];
```

Comment:
The problem I see now is that this option is available by default even though they don't use codeowners at all and/or don't use the GitHub groups in backstage. So this might need additional documentation or it could lead to confused users who create catalog files which can't be imported due to missing required fields.

Reply:
That's a fair point. A config flag would be a quick fix as well but introduce something that would need to be deprecated for a more automated solution. Let me think on this a little.